pages/lists/list-user/web-space/de.text

```diff
@@ -0,0 +1,13 @@
+@title = 'Web Space'
+(Gemeinsame Dokumente)
```

Comment:
this line won't appear in the title and should be removed or added to the headline below

Reply:
"this line" means line 2? It's intended not to show that in the title line, but in normal text size below the title. It's simply a more applicable wording than "web space". In the final view it should look like

### Web Space
(Gemeinsame Dokumente)

Reply:
it would look like this: https://share.riseup.net/#xCQWJBd0q7r5iOGci3qkXQ

Reply:
Hm, that is not intended, "(Gemeinsame Dokumente)" should appear below title and above list. Not below list.
My amber knowledge is too poor, can you shift "(Gemeinsame Dokumente)"  to the intended place directlybelow the title?

Reply:
Try: ```@summary = 'Gemeinsame Dokumente'```